PhysicsTools/SelectorUtils/python/tools/auxiliary_cff.py

```diff
@@ -0,0 +1,5 @@
+
+# Define constants for data formats for VID and anyone else who needs them
+class DataFormat:
```

Comment:
Hi @ikrav - please give this class a name that matches the python file (and given that this python file does not define a cms sequence, its more appropriate that it not include "_cff" in its name.